business_requirement_deliverable_categ/__openerp__.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# © 2016 Elico Corp (www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Business Requirement Deliverable Category",
+    'category': 'Business Requirements Management',
+    'summary': 'Business Requirement Deliverable Task Categories',
+    "version": "8.0.3.0.2",
+    "website": "www.elico-corp.com",
+    "author": "Elico corp",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": [
+        "business_requirement_deliverable",
+        "project_task_category",
```

Comment:
@dreispt tags are not report friendly. I do not like the m2m because you cannot filter/report properly.
with a m2o categ like this you can create clean reports for the task type allocated time vs spent.
It would be impossible to achieve with tags (see comment on the topic in #2)